cloudant-client/src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -48,35 +49,32 @@
 import java.util.List;
 import java.util.Map;
 
-@Category(RequiresDB.class)
-public class DatabaseTest {
+@RequiresDB
+public class DatabaseTest extends TestWithDb {
 
-    public static CloudantClientResource clientResource = new CloudantClientResource();
-    public static DatabaseResource dbResource = new DatabaseResource(clientResource);
+    @RegisterExtension
+    public static MockWebServerExtension mockWebServerExt = new MockWebServerExtension();
 
-    @ClassRule
-    public static RuleChain chain = RuleChain.outerRule(clientResource).around(dbResource);
-    @ClassRule
-    public static MockWebServer mockWebServer = new MockWebServer();
+    private static MockWebServer mockWebServer;
 
-    private static Database db;
-    private static CloudantClient account;
-
-    @BeforeClass
-    public static void setUp() throws Exception {
-        account = clientResource.get();
-        db = dbResource.get();
+    // TODO before class?

```

Comment:
Does this need doing?

Reply:
See 1d42697

Per the commit comment "mockWebServer needs to be fetched from extension before each test because the extension is a "per test" extension (overrides beforeEach/afterEach)".

It's a bit wasteful to stop/start the mock web server for each test when currently only one test uses it, but I think the overhead is negligible.